docs/apm/api.asciidoc

```diff
@@ -708,3 +709,87 @@ curl -X DELETE "http://localhost:5601/api/apm/sourcemaps/apm:foo-1.0.0-644fd5a9"
 --------------------------------------------------
 {}
 --------------------------------------------------
+
+////
+*******************************************************
+*******************************************************
+////
+
+[role="xpack"]
+[[agent-key-api]]
+=== Agent Key API
+
+The Agent Key API allows you to configure agent keys to authorize requests from agents to the APM Server.
+
+The following Agent key APIs are available:
+
+* <<apm-create-agent-key>> to create an agent key
+
+[float]
+[[use-agent-key-api]]
+==== How to use APM APIs
+
+.Expand for required headers, privileges, and usage details
+[%collapsible%closed]
+======
+include::api.asciidoc[tag=using-the-APIs]
+======
+
+////
+*******************************************************
+////
+
+[[apm-create-agent-key]]
+==== Create agent key
+
```

Comment:
Maybe here? One of the other APIs has an extended description here, so I think it works.

```suggestion
==== Create agent key

Create an APM agent key. Specify privileges in the request body at creation time. If privileges are not specified at creation time, the created key will have all privileges.

```